### ISSUE TITLE
Library usage example correction

### ISSUE THREAD

**Issue by Kooper** [flopp/go-staticmaps#14]
Hello,

Please correct the example of library usage:

``` diff
diff --git a/README.md b/README.md
index 5d994f0..e5a0fad 100644
--- a/README.md
+++ b/README.md
@@ -38,7 +38,7 @@ import (
 func main() {
   ctx := sm.NewContext()
   ctx.SetSize(400, 300)
-  ctx.AddMarker(sm.NewMarker(s2.LatLng{52.514536, 13.350151}, color.RGBA{0xff, 0, 0, 0xff}, 16.0))
+  ctx.AddMarker(sm.NewMarker(s2.LatLngFromDegrees(52.514536, 13.350151), color.RGBA{0xff, 0, 0, 0xff}, 16.0))

   img, err := ctx.Render()
   if err != nil {

```

**Comment by flopp** [comment]
Thanks for reporting the error!
I just fixed the example: f992a78f1c6956ab3c0241364b50272797fe29f2

May I add you to the list of contributors in README.md?

**Comment by Kooper** [comment]
Sure!

BTW thank you for the library! It is very easy to use.

I'm utilizing it to quickly overview geospatial data, returned by RESTful service. By specifying 'output=png' request parameter I get a nice picture of results on minimap, otherwise returned as JSON objects which aren't easy to recognize. The library works very well and integrates easily with HTTP service via Writer interface.
